### ISSUE TITLE
tunnels

### ISSUE THREAD

**Issue by eliboni** [protomaps/protomaps-leaflet#145]
Hi, in https://github.com/protomaps/protomaps-leaflet/tree/main/src/default_style/themes.ts I see several entries for tunnels, line `tunnel_medium: "#d5d5d5",` however in the out-of-the-box  PMTiles I can't seem to find an attributes (kind / kind_detail / etc) that allows to identify a way is a tunnel.

Is this a leftover from 1.24 to 2.0 or is there a way to understand if a road is a tunnel? I'd like if it were possible to render tunnels since in some cases  (like motorbike, bicycle) users  tend to avoid these if possible.

Thanks!

**Comment by bdon** [comment]
Yes, it's a leftover, I simply haven't had time to adapt the logic to handle those since it needs to be slightly different from MapLibre.

**Comment by eliboni** [comment]
ok  - great, from your answer it seems that soon or later the logic will be able handle this as well. thanks
